modules/apps/asset/asset-list-web/src/main/java/com/liferay/asset/list/web/internal/display/context/EditAssetListDisplayContext.java

```diff
@@ -1175,6 +1237,56 @@ private PortletURL _getAssetEntryItemSelectorPortletURL(
 			assetEntryItemSelectorCriterion);
 	}
 
+	private JSONArray _getAssetListEntrySegmentsEntryRelJSONArray() {
+		List<AssetListEntrySegmentsEntryRel>
+			assetListEntrySegmentsEntryRelList =
```

Comment:
Call it `assetListEntrySegmentsEntryRels`

Reply:
Renamed ;)